tests/default/config/bootstrap.php

```diff
@@ -1,11 +1,17 @@
 <?php
 
+use Dotenv\Dotenv;
 use staabm\PHPStanDba\QueryReflection\QueryReflection;
 use staabm\PHPStanDba\QueryReflection\RuntimeConfiguration;
 use staabm\PHPStanDba\Tests\ReflectorFactory;
 
 require_once __DIR__.'/../../../vendor/autoload.php';
 
+if (false === getenv('GITHUB_ACTION')) {
```

Comment:
I don't think we need this IF-case arround the loading.

another question: does the dotenv class automatically load the .env.dist file now?
I tried running this PR locally and I am still getting errors
```
$ composer test
> phpunit -c tests/default/config/phpunit.xml
PHPUnit 9.5.16 by Sebastian Bergmann and contributors.

Error in bootstrap script: RuntimeException:
Unknown mode:
PHP Warning:  Undefined array key "DBA_HOST" in C:\dvl\Workspace\phpstan-dba\tests\ReflectorFactory.php on line 29
PHP Warning:  Undefined array key "DBA_USER" in C:\dvl\Workspace\phpstan-dba\tests\ReflectorFactory.php on line 30
PHP Warning:  Undefined array key "DBA_PASSWORD" in C:\dvl\Workspace\phpstan-dba\tests\ReflectorFactory.php on line 31
PHP Warning:  Undefined array key "DBA_DATABASE" in C:\dvl\Workspace\phpstan-dba\tests\ReflectorFactory.php on line 32
PHP Warning:  Undefined array key "DBA_MODE" in C:\dvl\Workspace\phpstan-dba\tests\ReflectorFactory.php on line 33
PHP Warning:  Undefined array key "DBA_REFLECTOR" in C:\dvl\Workspace\phpstan-dba\tests\ReflectorFactory.php on line 34
```

Reply:
ahh I get it now. I need to copy the .env.dist file to .env once.. it works now

Reply:
yeah, the .dist file is supposed to be like a template for the envs..

But it's commonly used so some defaults / test setup can be blaced in dist and setup from the existing file..

Reply:
I see. I am used to a process where the .dist file is automtically picked up, without the need to copy it a priori.

but thats fine.

thanks for the work you put into it here. I will merge it.